src/components/Reports/WbsTable/WbsTable.jsx

```diff
@@ -39,4 +40,4 @@ export const WbsTable = ({ wbs, skip, take }) => {
       <div>{WbsList.length > 0 ? WbsList : <Stub />}</div>
     </div>
   );
-};
+}
```

Comment:
To fix the above lint issue, you'll also need to export `WbsTable` from this file. Please take this PR as an example - https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1215/files#diff-ac8689ba4b8e425618c02cb7d51230facb13753ef18963a66b2905722f6eefae